source/_includes/installation/operating_system.md

```diff
@@ -71,46 +77,119 @@ To boot Home Assistant OS, the BIOS needs to have UEFI boot mode enabled and Sec
+   - This means you will lose all the data as well as the previously installed operating system.
+   - Back up your data before continuing with the next step.
+2. Create a so called *live operating system* on a USB device:
+   - Follow the ([Ubuntu instructions](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)) on writing an Ubuntu iso file onto a USB device.
+3. Insert the USB device into the system on which you want to run Home Assistant.
+   - Boot the live operating system.
+   - The system then starts Ubuntu.
+4. In Ubuntu, open a browser and download the image to your computer.
+      - Copy the URL for the image.
+  
+      {% if site.installation.types[page.installation_type].variants.size > 1 %}
+      {% tabbed_block %}
+      {% for variant in site.installation.types[page.installation_type].variants %}
+
+      - title: {{ variant.name }}
+        content: |
+
+          ```text
+          {{release_url}}/{{site.data.version_data.hassos[variant.key]}}/haos_{{ variant.key }}-{{site.data.version_data.hassos[variant.key]}}.img.xz
+          ```
+
+          {% if variant.key == "odroid-n2" %}
+          [Guide: Flashing Odroid-N2 using OTG-USB](/hassio/flashing_n2_otg/)
+          {% elsif variant.key == "rpi4" or variant.key == "rpi3" %}
+            _(64-bit is recommended)_
+          {% endif %}
+
+      {% endfor %}
+      {% endtabbed_block %}
+      {% else %}
+
+      ```text
+      {% assign board_key = site.installation.types[page.installation_type].variants[0].key %}
+      {{release_url}}/{{site.data.version_data.hassos[board_key]}}/haos_{{ board_key }}-{{site.data.version_data.hassos[board_key]}}.img.xz
+      ```
+
+      {% endif %}
+
+      _Select and copy the URL or use the "copy" button that appear when you hover it._
 
-{% endif %}
+
+       - Paste the URL into your browser to start the download.
+
+
+      {% endif %}
+
+5. In Ubuntu, in the bottom left corner, select **Show Applications**.
+6. In the applications, search and open **Disks** and start restoring the HOAS image:
+   1. In **Disks**, select the external USB device.
+   2. On top of the screen, select the three dots menu and select **Restore Disk Image...**.
+      ![Restore disk image: select three dots menu](/images/installation/ubuntu_restore_disk_image.png)
+   3. Select the image you just downloaded.
+      ![Restore disk image: select three dots menu](/images/installation/select_haos.png)
+   4. Select **Start Restoring...**.
+      ![Restore disk image: select three dots menu](/images/installation/start_restoring.png)
+   5. Confirm by selecting **Restore**.
+      ![Restore disk image: select three dots menu](/images/installation/restore.png)
+   6. In the partitions overview, you should now see the **hassos-boot** partition.
+      ![Restore disk image: select three dots menu](/images/installation/hassos_boot_partition.png)
+      - Select the **Play** button.
```

Comment:
That is not necessary. The play button will mount that partition so you can edit things, but no touching/editing things required here.

You can ask the user to shutdown and remove the USB flash drive.

```suggestion
```